### ISSUE TITLE
RoborazziATFAccessibilityCheckOptions - Configuring supression results to an error

### ISSUE THREAD

**Issue by aileene-willow** [takahirom/roborazzi#613]
Using this configuration

`roborazziAccessibilityOptions = RoborazziATFAccessibilityCheckOptions(
                RoborazziATFAccessibilityChecker(
                    preset = AccessibilityCheckPreset.LATEST,
                    suppressions = matchesElements(withTestTag("suppress"))
                ),
                failureLevel = RoborazziATFAccessibilityChecker.CheckLevel.Error
            )`

returns this error:

Argument type mismatch: actual type is 'com.android.layoutlib.hamcrest.Matcher<com.google.android.apps.common.testing.accessibility.framework.AccessibilityViewCheckResult!>!', but 'org.hamcrest.Matcher<in com.google.android.apps.common.testing.accessibility.framework.AccessibilityViewCheckResult>' was expected.

**Comment by aileene-willow** [comment]
Hi @takahirom Yes, I am using those exact imports.  This is my full configuration:

import androidx.compose.ui.test.junit4.AndroidComposeTestRule
import androidx.compose.ui.test.onRoot
import com.dropbox.differ.SimpleImageComparator
import com.github.takahirom.roborazzi.AccessibilityCheckAfterTestStrategy
import com.github.takahirom.roborazzi.ExperimentalRoborazziApi
import com.github.takahirom.roborazzi.RoborazziATFAccessibilityCheckOptions
import com.github.takahirom.roborazzi.RoborazziATFAccessibilityChecker
import com.github.takahirom.roborazzi.RoborazziOptions
import com.github.takahirom.roborazzi.RoborazziRule
import com.google.android.apps.common.testing.accessibility.framework.AccessibilityCheckPreset
import com.google.android.apps.common.testing.accessibility.framework.AccessibilityCheckResultUtils.matchesElements
import com.google.android.apps.common.testing.accessibility.framework.matcher.ElementMatchers.withTestTag

```
@OptIn(ExperimentalRoborazziApi::class)
fun tmoRoborazziRule(
    // captureImage is not used for tests of pop-up modals
    composeTestRule: AndroidComposeTestRule<*, *>,
    captureImage: Boolean = true,
): RoborazziRule {
    return RoborazziRule(
        composeRule = composeTestRule,
        captureRoot = composeTestRule.onRoot(),
        options = RoborazziRule.Options(
            // Accessibility check options is only supported for SDK 34 and higher

            roborazziAccessibilityOptions = RoborazziATFAccessibilityCheckOptions(
                RoborazziATFAccessibilityChecker(
                    preset = AccessibilityCheckPreset.LATEST,


                   suppressions = matchesElements(withTestTag("suppress"))

                ),
                failureLevel = RoborazziATFAccessibilityChecker.CheckLevel.Error
            ),
            accessibilityCheckStrategy = AccessibilityCheckAfterTestStrategy(),
            outputDirectoryPath = "./src/test/roborazzi/images",
            captureType = if (captureImage) RoborazziRule.CaptureType.LastImage() else RoborazziRule.CaptureType.None,
            roborazziOptions = RoborazziOptions(
                compareOptions = RoborazziOptions.CompareOptions(
                    imageComparator = SimpleImageComparator(maxDistance = 0.014F, hShift = 1)
                )
            )
        ),
    )
}

**Comment by takahirom** [comment]
Thanks. Could you check the version of accessibility framework?
It seems Roborazzi uses this.` com.google.android.apps.common.testing.accessibility.framework:accessibility-test-framework:4.1.1`.

**Comment by aileene-willow** [comment]
No longer an issue :) Importing com.google.android.apps.common.testing.accessibility.framework:accessibility-test-framework:4.1.1 separately worked! Thank you!!
